README.md

```diff
@@ -80,16 +80,17 @@ Alternatively, you could pass the webhook as a configuration option.
+| Option                 | Description                                                                                                                                                                                                                                     | Default                                                      |
+| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ |
+| `notifyOnSuccess`      | Determines if a succesfull release should trigger a slack message to be sent. If `false` this plugin does nothing on success.                                                                                                                   | false                                                        |
+| `notifyOnFail`         | Determines if a failed release should trigger a slack message to be sent. If `false` this plugin does nothing on fail.                                                                                                                          | false                                                        |
+| `onSuccessTemplate`    | Provides a template for the slack message object on success when `notifyOnSuccess` is `true`. See [templating](#templating).                                                                                                                    | undefined                                                    |
+| `onFailTemplate`       | Provides a template for the slack message object on fail when `notifyOnFail` is `true`. See [templating](#templating).                                                                                                                          | undefined                                                    |
+| `markdownReleaseNotes` | Pass release notes through markdown to slack formatter before rendering.                                                                                                                                                                        | false                                                        |
+| `slackWebhookEnVar`    | Environment variable to use to select the default Slack webhook value.                                                                                                                                                                          | SLACK_WEBHOOK                                                |
+| `slackWebhook`         | Slack webhook created when adding app to workspace.                                                                                                                                                                                             | value of the environment variable matching slackWebhookEnVar |
```

Comment:
I like this explanation more than the old one 👍

Reply:
I think it looks better to have `slackWebhookEnVar` as an inline code block. Like this:

```suggestion
| `slackWebhook`         | Slack webhook created when adding app to workspace.                                                                                                                                                                                             | value of the environment variable matching `slackWebhookEnVar` |
```

What do you think?